newscoop/src/Newscoop/NewscoopBundle/Controller/EmbedlyController.php

```diff
@@ -78,7 +78,7 @@ public function Process($parameters)
             $decoder = new JsonDecode(true);
             $parameters['response'] = $decoder->decode($json, 'json');
         } else {
-            throw new Exception('Something went wrong');
+            throw new \Exception('Something went wrong');
```

Comment:
I will get the message from the response that clearly says what's the error. In aes it wont be displayed in the toaster msg because we use there diffrent messages, but in the response it will be clear for devs, when it will fail.